classes/editor_ajax.php

```diff
@@ -65,7 +65,8 @@ public function getLatestLibraryVersions() {
 
         return $DB->get_records_sql("
             SELECT hl4.id, hl4.machine_name, hl4.title, hl4.major_version,
-                hl4.minor_version, hl4.patch_version, hl4.has_icon, hl4.restricted
+                hl4.minor_version, hl4.patch_version, hl4.has_icon, hl4.restricted,
+                0 AS patch_version_in_folder_name
```

Comment:
What does this do, im not seeing this field name referenced anywhere in the PR?

Reply:
This is to fix up a key doesn't exist warning here in the editor submodule https://github.com/h5p/h5p-editor-php-library/blob/53dc7bdc57b17f5a0d55a8871a36d1b1bd631e70/h5peditor.class.php#L633